backend/src/zac/forms/api/views.py

```diff
@@ -0,0 +1,38 @@
+from drf_spectacular.utils import extend_schema
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from ..client import OpenFormsClient
+
+
+class OpenFormsFormListView(APIView):
```

Comment:
let's avoid confusion with the real Open Forms and express clearly this is for the prototype

```suggestion
class OpenFormsProtoFormListView(APIView):
```

Reply:
As discussed: use generic terminology instead.